src/header/navigation.rs

```diff
@@ -81,16 +81,11 @@ pub fn Header() -> impl IntoView {
             ..Default::default()
         },
         NavEntry {
-            href: "#".to_string(),
-            text: "More".to_string(),
-            icon: NavIcon::More,
+            href: "/broadcast".to_string(),
```

Comment:
Can we make this go directly to `broadcast/transaction` or `send/transaction`?

That way the Transaction tab becomes highlighted.